composer.json

```diff
@@ -30,5 +30,11 @@
         "psr-4": {
             "Phergie\\Tests\\Plugin\\Http\\": "tests"
         }
+    },
+    "config": {
+        "sort-packages": true,
+        "platform": {
+          "php": "7.6.19"
```

Comment:
Is this version number a typo?

Reply:
oh snap, my eyes were crossed and missed this, good catch @elazar

Reply:
No, just a random valid one above the lower versions to ensure we don't run into odd packages with odd minimum version requirements